src/govuk/all.js

```diff
@@ -50,6 +51,11 @@ function initAll (options) {
   var $toggleButton = scope.querySelector('[data-module="govuk-header"]')
   new Header($toggleButton).init()
 
+  var $notificationBanners = scope.querySelectorAll('[data-module="govuk-notification-banner"]')
```

Comment:
We could just initialise the first instance of the component here. This might also remove the need to check in the JavaScript whether there's an already initialised notification banner on the page. However:
1. The initialisation is tied to the module, not the auto-focus behaviour so in that sense maybe we should keep the two separate and initialise all notification banners (although we haven't done this for the error summary but then that is less likely to have duplicates on the page)
2. We don't control how teams initialise components so they could initialise them individually and for that reason we should maybe have the check in JS. 
Thoughts welcome.

Reply:
I thinks this has been resolved by https://github.com/alphagov/govuk-frontend/pull/1994#discussion_r512074290